src/components/media/Media.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../styles/variables';
+
+// the outermost element
+.bdl-Media {
+    align-items: flex-start;
+    display: flex;
+}
+
+// the media content, ie, an avatar
+.bdl-Media-figure {
+    align-self: flex-start;
+    flex-grow: 0;
```

Comment:
We usually want to define all three properties, since otherwise we're leaving the basis/width up to the various browsers. Is this `flex: 0 0 auto;`?

Reply:
The intersection of the shorthand with IE and minifiers is kind of ambiguous but I will give the shorthand a shot

https://developer.mozilla.org/en-US/docs/Web/CSS/flex#Syntax